ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -266,14 +266,19 @@ export const ConceptHomepage = withCurrentWorkspace()(
       }
     }
 
-    triggerSearch(e) {
+    handleChange(e) {
+      this.setState({currentSearchString: e});
+    }
+
+    handleSubmit(e) {
       // search on enter key
-      if (e.keyCode === 13) {
-        const searchTermLength = e.target.value.trim().length;
+      if (e.key === 'Enter' ) {
```

Comment:
(Opt) Bare strings always scare me. Either here or as a follow-on, we could use https://github.com/nfriend/ts-key-enum to refer to these keys in a type-safe way.

Reply:
running `npm audit fix` shows a few breaking changes (not yet applied)

```
+ @angular-devkit/build-angular@0.803.17
added 204 packages from 111 contributors, removed 164 packages, updated 76 packages and moved 6 packages in 33.769s
fixed 13 of 78 vulnerabilities in 67513 scanned packages
  1 vulnerability required manual review and could not be updated
  3 package updates for 64 vulnerabilities involved breaking changes
  (use `npm audit fix --force` to install breaking changes; or refer to `npm audit` for steps to fix these manually)

```